README.md

```diff
@@ -202,15 +202,35 @@ new_elevator_data = ClearanceItemCreateData(
     ControllerID=5000,
     ControllerClassType="SoftwareHouse.NextGen.Common.SecurityObjects.iStarController"
 )
-response = ccure_api.clearance_item.create(cit.ELEVATOR, create_data=new_elevator_data)
+response = ccure.clearance_item.create(ObjectType.ELEVATOR.complete, create_data=new_elevator_data)
 ```
 
 #### Delete ClearanceItem
 
 ```python
 from acslib import CcureAPI
-from acslib.ccure.types import ClearanceItemType as cit
+from acslib.ccure.types import ObjectType
+
 # delete a door
-ccure_api = CcureAPI()
-response = ccure_api.clearance_item.delete(cit.DOOR, 5000)
+ccure = CcureAPI()
+response = ccure.clearance_item.delete(ObjectType.DOOR.complete, 5000)
+```
+
+### Other item types
+
+#### Search for CCure item
+
+```python
+from acslib import CcureAPI
+from acslib.ccure.filters import CcureFilter, NFUZZ
+
+# search for schedule objects by ObjectID
+ccure = CcureAPI()
+schedule_type_full = "SoftwareHouse.CrossFire.Common.Objects.TimeSpec"
+search_filter = CcureFilter()
+response = ccure.ccure_object.search(
+    object_type=schedule_type_full,
+    search_filter=search_filter,
+    terms=[5001]
+)
```

Comment:
I didn't add any support specifically for schedule objects, but users can always do CRUD operations on whatever types they want if they have something weird like this